lesson01/hotel/src/com/andersenhotels/view/common/View.java

```diff
@@ -0,0 +1,38 @@
+package com.andersenhotels.view.common;
+
+/**
+ * The View interface defines the basic methods for user interaction
+ * in the hotel management application. Implementing classes should
+ * provide concrete behaviors for each method.
+ */
+public interface View {
```

Comment:
good solution 👍 
In the future you can create a new impl for this interface and just reuse the basic logic)

Reply:
Thank you!